Destination/AbstractDestination.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Destination;
+
+use Psr\Log\LoggerInterface;
+use Zenstruck\BackupBundle\Rotator\Rotator;
+
+abstract class AbstractDestination implements Destination
```

Comment:
Again, it is a coding style. I prefer to have prefix/suffix when it comes to abstract classes, interfaces and traits.

Your proposal is fine, I will change the name.

Sent from my iPad

> On Sep 8, 2015, at 5:54 PM, Kevin Bond <notifications@github.com> wrote:
> 
> In Destination/AbstractDestination.php:
> 
> > @@ -0,0 +1,170 @@
> > +<?php
> > +
> > +namespace Zenstruck\BackupBundle\Destination;
> > +
> > +use Psr\Log\LoggerInterface;
> > +use Zenstruck\BackupBundle\Rotator\Rotator;
> > +
> > +abstract class AbstractDestination implements Destination
> I try not using Abstract to prefix abstract classes if I can avoid it. Can we call this something else?
> 
> What about an abstract class called RotatingDestination instead of the interface? (not sure if I like not using an interface though...)
> 
> —
> Reply to this email directly or view it on GitHub.
>